loopchain/channel/channel_inner_service.py

```diff
@@ -490,37 +492,46 @@ def __add_tx_list(self, tx_list):
 
             new_block_height = subscriber_block_height + 1
             new_block = blockchain.find_block_by_height(new_block_height)
-            confirm_info: str = blockchain.find_confirm_info_by_height(new_block_height).decode("utf-8")
+            confirm_info: bytes = blockchain.find_confirm_info_by_height(new_block_height)
```

Comment:
Is confirm_info `bytes` or `bytearray`?
I suggest to change `find_confirm_info_by_height` return bytes.

Reply:
Updated to bytes!